tests/test_template.py

```diff
@@ -30,18 +24,8 @@ class TemplateTest(unittest.TestCase):
     """Tests for the template security feature."""
 
     def testPrint(self):
-        if PY2:
-            tpl = "$code:\n    print 'blah'"
-            # print_function has been imported from __future__ so the print statement doesn't exist anymore
-            self.assertRaises(SyntaxError, t, tpl)
-        else:
-            tpl = "$code:\n    print('blah')"
-            self.assertRaises(NameError, t(tpl))
-
-    def testRepr(self):
```

Comment:
Why drop __testRepr()__?

Reply:
Because it doesn't test py3 at all. The original code:

```
    def testRepr(self):
        if PY2:  # this feature doesn't exist in Py3 anymore
            tpl = "$code:\n    `1`"
            self.assertRaises(SecurityError, t, tpl)
```